# for local planner
max_velocity: 0.25
min_velocity: 0.1
max_rotation: 0.6
control_frequency: 8
acceleration: 1.02
rotation_tolerance: 0.16
proportion: 1.8 #1.5
integration: 2050
wait_after_qr: 6.4
turnning_at_boot: True
odom_topic: '/odom_filtered'
imu_topic: '/imu'

# for lane detection
image_topic: '/usb_cam/image_raw'
show_image: False

# for qr reader script
wait_for_qr: 2.0
show_qr_image: False
qr_image_topic: '/usb_cam2/image_raw'
odom: '/odom_filtered'
wait_after_the_thirdqr: 5.0